contracts/options.cairo

```diff
@@ -185,6 +186,25 @@ func mint_option_token{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_che
                 strike_price=strike_price,
             );
         }
+
+        with_attr error_message("trade exceeds max liqpool deposit size"){
+            let (user_address) = get_caller_address();
+            let (optsize) = IOptionToken.balanceOf(contract_address=option_token_address, account=user_address);
```

Comment:
I wouldn't block the amount of option tokens the user has... just the amount the user mints at one go... cause that is the impact on the AMM